tests/integration/golang/admin/namespace/update_test.go

```diff
@@ -0,0 +1,150 @@
+//go:build integration
+
+package namespace
+
+import (
+	"context"
+	"net/http"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/suite"
+
+	"github.com/G-Research/fasttrackml/pkg/api/mlflow/common"
+	"github.com/G-Research/fasttrackml/pkg/api/mlflow/dao/models"
+	"github.com/G-Research/fasttrackml/pkg/ui/admin/request"
+	"github.com/G-Research/fasttrackml/tests/integration/golang/helpers"
+)
+
+type UpdateNamespaceTestSuite struct {
+	suite.Suite
+	helpers.BaseTestSuite
+}
+
+func TestUpdateNamespaceTestSuite(t *testing.T) {
+	suite.Run(t, new(UpdateNamespaceTestSuite))
+}
+
+func (s *UpdateNamespaceTestSuite) SetupTest() {
+	s.BaseTestSuite.SetupTest(s.T())
+}
+
+func (s *UpdateNamespaceTestSuite) Test_Ok() {
+	defer func() {
+		assert.Nil(s.T(), s.NamespaceFixtures.UnloadFixtures())
+	}()
+	_, err := s.NamespaceFixtures.CreateNamespace(context.Background(), &models.Namespace{
+		ID:                  1,
+		Code:                "default",
+		DefaultExperimentID: common.GetPointer(int32(0)),
+	})
+	assert.Nil(s.T(), err)
+	ns, err := s.NamespaceFixtures.CreateNamespace(context.Background(), &models.Namespace{
+		ID:                  2,
+		Code:                "test2",
+		Description:         "test namespace 2 description",
+		DefaultExperimentID: common.GetPointer(int32(0)),
+	})
+	assert.Nil(s.T(), err)
+
+	request := request.Namespace{
+		Code:        "test2Updated",
+		Description: "test namespace 2 description updated",
+	}
+	assert.Nil(
+		s.T(),
+		s.AdminClient.WithMethod(
+			http.MethodPut,
+		).WithRequest(
+			request,
+		).DoRequest("/namespaces/%d", ns.ID),
+	)
+
+	namespaces, err := s.NamespaceFixtures.GetNamespaces(context.Background())
```

Comment:
since we just test/use a particular(one) namespace, won't it be reasonable to get one namespace like GetNamespaceByID and check its updated values?